edgexfoundry/device-rest-go
# build(deps): Bump sdk and core-contracts to latest version

Signed-off-by: Chris Hung <chris@iotechsys.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rest-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. Run device-rest from this branch
2. verify there's no 404 not found error in metadata and devices are successfully added

## New Dependency Instructions (If applicable)
